src/main/java/uk/gov/hmcts/darts/dets/service/impl/DetsApiServiceImpl.java

```diff
@@ -65,4 +81,60 @@ public UUID saveBlobData(BinaryData binaryData) {
         client.upload(binaryData);
         return uniqueBlobId;
     }
+
+    @Override
+    public boolean deleteBlobDataFromContainer(UUID blobId) throws AzureDeleteBlobException {
+        try {
+            BlobServiceClient serviceClient = blobServiceFactory.getBlobServiceClientWithSasEndpoint(configuration.getSasEndpoint());
+            BlobContainerClient containerClient = blobServiceFactory.getBlobContainerClient(configuration.getContainerName(), serviceClient);
+            BlobClient client = blobServiceFactory.getBlobClient(containerClient, blobId);
+            Response<Boolean> response = client.deleteIfExistsWithResponse(DeleteSnapshotsOptionType.INCLUDE, null,
+                                                                           Duration.of(
```

Comment:
maybe config should just be duration, and use 60s or 1m in application.yaml

Reply:
This was copied from Marios ticket which has now been merged so will look into this.